src/features/auth/components/SignIn.tsx

```diff
@@ -66,6 +71,15 @@ export const SignIn = ({
 
   const isError = hasAttemptedSubmit && !isEmailValid;
 
+  useEffect(() => {
```

Comment:
whats the context here?

Reply:
also, this definitely isnt the way we should be caching it – shouldn't we cache emails used for google sign in as well?

best to revert this change; also lets both make different branches before pushing to staging

only listing and rq changes should be committed directly here